### PR TITLE
Update spanish translation

### PR DESCRIPTION
**Pull request by tadeu2**
I have fixed with new typos the PR  #6115 reverted in #6164 by @ElectricalBoy. Thanks.

### CODE REVIEW

File: localization/i18n/es/OrcaSlicer_es.po [hunk 1]
```diff
@@ -11888,14 +11903,14 @@ msgid ""
...
-"El área de relleno se amplía ligeramente para solaparse con la pared y "
+"El área de relleno se amplía ligeramente para solaparse con la perímetro y "
 "mejorar la adherencia. El valor porcentual es relativo a la anchura de línea "
 "del de baja densidad. Ajuste este valor a ~10-15% para minimizar la "
 "sobreextrusión potencial y la acumulación de material que resulta en "
 "superficies superiores ásperas."
 
 msgid "Top/Bottom solid infill/wall overlap"
-msgstr "Relleno sólido superior/inferior/solapamiento de paredes"
+msgstr "Relleno sólido superior/inferior/solapamiento de perímetroes"
```
Comment (tadeu2) [inline localization/i18n/es/OrcaSlicer_es.po:11913]: Perímetros

File: localization/i18n/es/OrcaSlicer_es.po [hunk 2]
```diff
@@ -12046,7 +12116,7 @@ msgid "Small area flow compensation (beta)"
 msgstr "Compensación de flujo en áreas pequeñas (beta)"
 
 msgid "Enable flow compensation for small infill areas"
-msgstr "Permitir la compensación de flujo en zonas con poco relleno"
+msgstr "Permitir la compensación de flujo en pequeñas zonas de relleno"
```
Comment (tadeu2) [inline localization/i18n/es/OrcaSlicer_es.po:12119]: Revert this

File: localization/i18n/es/OrcaSlicer_es.po [hunk 3]
```diff
@@ -13027,18 +13108,21 @@ msgid ""
...
 "Nota: Con el protector contra corrientes de aire activo, la falda se "
-"imprimirá a la distancia del faldón del objeto. Por lo tanto, si los bordes "
-"están activos, puede cruzarse con ellos. Para evitarlo, aumente el valor de "
-"la distancia de la falda.\n"
+"imprimirá a la distancia especificada en \"Distancia de falda\" del objeto. "
+"Por lo tanto, si los bordes están activos, puede cruzarse con ellos. Para "
+"evitarlo, aumente el valor de la \"Distancia de la falda\".\n"
+"imprimirá a la distancia especificada en \"Distancia de falda\" del objeto. "
+"Por lo tanto, si los bordes están activos, puede cruzarse con ellos. Para "
+"evitarlo, aumente el valor de la \"Distancia de la falda\".\n"
```
Comment (tadeu2) [inline localization/i18n/es/OrcaSlicer_es.po:13125]: Distancia de Falda

File: localization/i18n/es/OrcaSlicer_es.po [hunk 4]
```diff
@@ -14815,7 +14903,8 @@ msgstr "El resultado del test fallido se ha descartado."
 
 msgid "Flow Dynamics Calibration result has been saved to the printer"
 msgstr ""
-"El resultado de la Calibración de Dinámicas de Flujo se ha salvado en la "
+"El resultado de la Calibración de Dinámicas de Flujo se ha salvado en la El "
```
Comment (tadeu2) [inline localization/i18n/es/OrcaSlicer_es.po:14906]: Repetido, corregir

File: localization/i18n/es/OrcaSlicer_es.po [hunk 5]
```diff
@@ -15173,7 +15270,7 @@ msgstr "%s no es compatible con %s"
 msgid "TPU is not supported for Flow Dynamics Auto-Calibration."
 msgstr ""
 "El filamento TPU no está soportado para la Auto-Calibración de Dinámicas de "
-"Flujo."
+"Flujo.Flujo."
```
Comment (tadeu2) [inline localization/i18n/es/OrcaSlicer_es.po:15273]: Repetido

File: localization/i18n/es/OrcaSlicer_es.po [hunk 6]
```diff
@@ -15212,7 +15309,9 @@ msgstr "Éxito recuperando los resultados históricos"
 
 msgid "Refreshing the historical Flow Dynamics Calibration records"
 msgstr ""
-"Refrescar el histórico de resultados de Calibración de Dinámicas de Flujo"
+"Refrescar el histórico de resultados de Calibración de Dinámicas de "
```
Comment (tadeu2) [inline localization/i18n/es/OrcaSlicer_es.po:15312]: Repetido

File: localization/i18n/es/OrcaSlicer_es.po [hunk 7]
```diff
@@ -16586,16 +16684,16 @@ msgstr "¡Se ha conectado correctamente a SimplyPrint!"
...
 msgstr "Error interno"
 
 msgid "Unknown error"
 msgstr "Error desconocido"
 
 msgid "SimplyPrint account not linked. Go to Connect options to set it up."
 msgstr ""
-"La cuenta SimplyPrint no está vinculada. Vaya a Opciones de conexión para "
-"configurarla."
+"Error desconocidoLa cuenta SimplyPrint no está vinculada. Vaya a Opciones de "
```
Comment (tadeu2) [inline localization/i18n/es/OrcaSlicer_es.po:16695]: Separar y punto

File: resources/web/data/text.js [hunk 8]
```diff
@@ -323,8 +323,8 @@ var LangText = {
     t111: "Crear nuevo",
     t112: "Unirse al programa",
     t113: "Puede cambiar su elección en preferencias en cualquier momento.",
-    orca1: "Edit Project Info",
-    orca2: "no model information",
+    orca1: "Editar información del proyecto",
+    orca2: "no hay información sobre el modelo",
```
Comment (tadeu2) [inline resources/web/data/text.js:327]: No

**Review by tadeu2 — COMMENTED** [review]
All fixed

**Review by SoftFever — APPROVED** [review]
Thank you

**Pull request merged**